### ISSUE TITLE
auto-indent is broken

### ISSUE THREAD

**Issue by alexandernst** [atom/language-json#20]
I have this JSON:

``` json
    {
      "name": "angular-multi-select",
      "version": "5.0.1",
      "description": "A multi select dropdown directive for AngularJS",
      "main": [
        "angular-multi-select.js",
        "angular-multi-select.css"
      ],
      "repository": {
        "type": "git",
        "url": "git@github.com:alexandernst/angular-multi-select.git"
      },
      "dependencies": {
        "angular": "1.3.15",
        "angular-mocks": "1.3.15",
        "angular-filter": "0.5.4",
        "grunt-contrib-uglify": "0.9.1",
        "grunt-contrib-jshint": "0.11.2",
        "grunt-contrib-jasmine": "0.8.2"
        },
      "keywords": [
        "angular",
        "dropdown",
        "multi",
        "select",
        "multiselect"
      ],
      "author": "Alexander Nestorov <alexandernst@gmail.com> (https://github.com/alexandernst)",
      "license": "MIT",
      "bugs": {
        "url": "https://github.com/alexandernst/angular-multi-select/issues"
      },
      "homepage": "https://github.com/alexandernst/angular-multi-select"
    }
```

And after I select all and hit `Ctrl + Alt + P` and write `auto indent`, Atom converts it to:

``` json
    {
        "name": "angular-multi-select",
        "version": "5.0.1",
        "description": "A multi select dropdown directive for AngularJS",
        "main": [
            "angular-multi-select.js",
            "angular-multi-select.css"
            ],
            "repository": {
                "type": "git",
                "url": "git@github.com:alexandernst/angular-multi-select.git"
                },
                "dependencies": {
                    "angular": "1.3.15",
                    "angular-mocks": "1.3.15",
                    "angular-filter": "0.5.4",
                    "grunt-contrib-uglify": "0.9.1",
                    "grunt-contrib-jshint": "0.11.2",
                    "grunt-contrib-jasmine": "0.8.2"
                    },
                    "keywords": [
                        "angular",
                        "dropdown",
                        "multi",
                        "select",
                        "multiselect"
                        ],
                        "author": "Alexander Nestorov <alexandernst@gmail.com> (https://github.com/alexandernst)",
                        "license": "MIT",
                        "bugs": {
                            "url": "https://github.com/alexandernst/angular-multi-select/issues"
                            },
                            "homepage": "https://github.com/alexandernst/angular-multi-select"
                        }
```

**Comment by 50Wliu** [comment]
Works fine for me on Atom 1.15.0-dev-d5785a1, so closing.  Please comment here if it's still an issue.

**Comment by alexandernst** [comment]
Please don't close issues unless they are fixed is stable. You're testing this with a version of Atom that won't reach stable for another 3 or 4 months.

And btw, no, this still isn't fixed in 1.13

**Comment by Ben3eeE** [comment]
@alexandernst Are you reproducing this when starting Atom using `atom --safe`. Is the language mode set to `JSON`?

I was unable to reproduce this using `Atom 1.13.0` here is what I did:

1. Opened Atom in safe mode using `atom --safe` from `cmd.exe`
1. Copied the json from your example and saved it as `test.json`
1. Ensured that the language mode indicated in the lower right corner on the status bar was set to `JSON`
1. Selected all text using <kbd>Ctrl-a</kbd>
1. Opened the `Command palette` using <kbd>Ctrl-Shift-P</kbd>
1. Ran the `editor:auto-indent` command from the command palette
1. **Expected** and **Actual**: The curly bracket on line 20 got outdented because it is on the wrong indentation level.

Is there anything I am missing to reproduce this issue?
